Documentation/ProjectFile/properties/property/CapillaryPressureVanGenuchten/t_regularization.md

```diff
@@ -0,0 +1,3 @@
+An optional input that indicates whether the capillary pressure is regularized
```

Comment:
Just a possibility for now, don't know what others think. @Scinopode ?
What reads better:
```xml
<property>
    <type>CapillaryPressureVanGenuchtenRegularized</type>
    <!-- other settings -->
</property>
```
or 
```xml
<property>
    <type>CapillaryPressureVanGenuchten</type>
    <regularization>true</regularization>
    <!-- other settings -->
</property>
```
My preference is, of course, the extra property `CapillaryPressureVanGenuchtenRegularized`.